chat_shell/chat_shell/services/sandbox_attachment_sync.py

```
@@ -0,0 +1,349 @@
+    request.prompt = _failed_attachment_prompt(request.prompt, failed)
+
+
+async def _file_already_synced(
+    sandbox: Any, path: str, expected_size: int | None
+) -> bool:
+    """Return whether a sandbox file already exists with the expected size."""
+    if not expected_size or expected_size < 0:
+        return False
+    try:
+        file_info = await sandbox.files.get_info(path)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)